`make install` fails

Installing `swift-graphql` (either through homebrew or `make install`) currently fails to build:

```
/private/tmp/SwiftGraphQL-20220315-23016-185y7qo/swift-graphql-2.3.0/.build/checkouts/SwiftFormat/Sources/CommandLine.swift:556:44: error: 'temporaryDirectory' is only available in macOS 10.12 or newer
                return FileManager.default.temporaryDirectory
```

--- a/Package.swift
+++ b/Package.swift
@@ -33,7 +33,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/JohnSundell/Files", from: "4.0.0"),
-        .package(url: "https://github.com/nicklockwood/SwiftFormat", from: "0.41.2"),
+        .package(url: "https://github.com/nicklockwood/SwiftFormat", .exact("0.49.4")),
         .package(url: "https://github.com/apple/swift-argument-parser", from: "1.0.0"),
         .package(url: "https://github.com/jpsim/Yams.git", from: "4.0.4"),
     ],

